tests/integration_tests/bugs/test_lp1898997.py

```diff
@@ -0,0 +1,67 @@
+"""Integration test for LP: #1898997
+
+cloud-init was incorrectly excluding Open vSwitch bridge members from its list
+of interfaces.  This meant that instances which had only one interface which
+was in an Open vSwitch bridge would not boot correctly: cloud-init would not
+find the expected physical interfaces, so would not apply network config.
+
+netplan has since been SRU'd so this issue does not reproduce exactly as it did
+previously: instead we use this integration test to confirm that cloud-init's
+state in a booted, Open vSwitch-enabled system matches what we expect.
+"""
+import pytest
+
+MAC_ADDRESS = "de:ad:be:ef:12:34"
+
+
+NETWORK_CONFIG = """\
+bridges:
+        ovs-br:
+            dhcp4: true
+            interfaces:
+            - enp5s0
+            macaddress: 52:54:00:d9:08:1c
+            mtu: 1500
+            openvswitch: {{}}
+ethernets:
+    enp5s0:
+      mtu: 1500
+      set-name: enp5s0
+      match:
+          macaddress: {}
+version: 2
+""".format(MAC_ADDRESS)
+
+
+@pytest.mark.lxd_config_dict({
+    "user.network_config": NETWORK_CONFIG,
+    "volatile.eth0.hwaddr": MAC_ADDRESS,
+})
+@pytest.mark.lxd_vm
+@pytest.mark.sru_2020_11
+@pytest.mark.ubuntu
+class TestInterfaceListingWithOpenvSwitch:
+    def test_ovs_member_interfaces_not_excluded(self, client):
```

Comment:
After the changes to pycloudlib, this test is now not passing/failing as expected; I need to debug the issue.

Reply:
This was a simple typo, thankfully!